cumulus/client/consensus/aura/src/collators/lookahead.rs

```diff
@@ -95,6 +95,11 @@ pub struct Params<BI, CIDP, Client, Backend, RClient, CHP, Proposer, CS> {
 	pub authoring_duration: Duration,
 	/// Whether we should reinitialize the collator config (i.e. we are transitioning to aura).
 	pub reinitialize: bool,
+	/// This is an experimental cli arg, it is meant to be used only if collator is overshooting
+	/// the PoV size, and building blocks that do not fit in the max_pov_size.
+	///
+	/// It will be removed once https://github.com/paritytech/polkadot-sdk/issues/6020 is fixed.
+	pub experimental_max_pov_size: Option<u32>,
```

Comment:
Should it be percent based? 100 meaning full pov size. Nobody should ever use more than that. Not a strong opinion on it though.

Reply:
Done.